src/resources/js/views/multiday-events.js

```diff
@@ -77,11 +77,10 @@ tribe.events.views.multidayEvents = {};
 	 * @return {jQuery} jQuery object of visible multiday event or false if none found
 	 */
 	obj.findVisibleMultidayEvents = function( $container, $hiddenMultidayEvent ) {
-		var eventId = $hiddenMultidayEvent.data( 'event-id' );
+		var eventId = $hiddenMultidayEvent.parent().data( 'event-id' );
```

Comment:
i'd like to use the `closest()` function, as if the structure of the HTML changes, this won't work anymore.